src/main/java/com/thealgorithms/maths/QuadraticEquation.java

```diff
@@ -0,0 +1,28 @@
+package com.thealgorithms.maths;
+
+public class QuadraticEquation {
+    static double[] solve(double a, double b, double delta) {
```

Comment:
Please remove console logging and return the appropriate number of roots in an array instead. Return an empty array if there're no real roots.